vignettes/xcms.Rmd

```diff
@@ -1285,6 +1284,141 @@ identification of e.g. features with significant different
 intensities/abundances it is suggested to use functionality provided in other R
 packages, such as Bioconductor's excellent `limma` package.
 
+## Alignment to an external reference dataset
+
+In certain experiments, aligning two different datasets is necessary. This
+can involve comparing runs of the same samples conducted across different
+laboratories or runs with MS2 recorded after the initial MS1 run. Across
+laboratories and over time, the same samples may result in variation in
+retention time, especially because the LC system can be quite unstable. In these
+cases, an alignment step using the `adjustRtime()` function with the
+`LamaParam` parameter can allow the user to perform this type of alignment.
+We will go through this step by step below. 
+
+Let's load an already analyzed dataset `ref` and our previous dataset before
+alignment, which will be `tst`. We will first restrict their retention time
```

Comment:
I'm wondering - do we need to have the same retention time range? I think we just did that for the unit test? if we do that here it sounds like this is a prerequisite.